React/CoreModules/RCTAlertController.m

```diff
@@ -35,6 +35,9 @@ - (void)show:(BOOL)animated completion:(void (^)(void))completion
 
 - (void)hide
 {
+  [_alertWindow resignKeyWindow];
```

Comment:
There's a recommendation that we shouldn't call this directly on [Apple docs](https://developer.apple.com/documentation/uikit/uiwindow/1621618-resignkeywindow)

Reply:
Ugh, my bad.  I guess I made an invalid assumption that UIWindow was like NSWindow